modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -52,6 +52,8 @@ public class UpgradeCatchAllCheck {
 		fDSTableSchemaBuilder.add("test123", "test123");
 
 		_htmlParser.extractText(null);
+
+		new ArrayList<>(null);
```

Comment:
@MicaelleSilvaa each pattern that adds a new import must add the import on the expected file as well.

Reply:
resolved